sensing/pointcloud_preprocessor/CMakeLists.txt

```diff
@@ -76,6 +78,7 @@ ament_auto_add_library(pointcloud_preprocessor_filter SHARED
   src/pointcloud_accumulator/pointcloud_accumulator_nodelet.cpp
   src/vector_map_filter/lanelet2_map_filter_nodelet.cpp
   src/distortion_corrector/distortion_corrector.cpp
+  src/distortion_corrector/undistorter.cpp
```

Comment:
What is the difference between the roles of distortion_corrector and undistorter?

Reply:
Sorry for the late reply, 
distortion_corrector is the node and undistorter is just a class that did all of the calculations.
Should I modify the name?

Reply:
I add the undistorter to the library in this commit https://github.com/autowarefoundation/autoware.universe/pull/7137/commits/8e8dfb968d6a4c0d2183e9e457b87fd3662e46bc

Reply:
I think I misunderstood the comment, I change the name from 
distortion_corrector -> distortion_corrector_nodelet
undistorter -> distortion_corrector

Reply:
sorry for late reply. I see :+1: